great_expectations/checkpoint/actions.py

```diff
@@ -165,6 +161,14 @@ class (for open source Great Expectations) or the GeCloudIdentifier (from Great
         """  # noqa: E501
         return NotImplementedError
 
+    # NOTE: To be promoted to 'run' after V1 development
+    def v1_run(self, checkpoint_result: CheckpointResult) -> None:
+        return self._v1_run(checkpoint_result=checkpoint_result)
```

Comment:
Will this method ever do more than directly wrap the private method? If not, why even have the private method?